.travis.yml

```diff
@@ -43,10 +43,15 @@ matrix:
   - env: "MODE=browserstack_optional"
 
 install:
-  - npm install
+  - yarn install
 
 before_script:
   - mkdir -p $LOGS_DIR
+  - sudo apt-key adv --keyserver pgp.mit.edu --recv D101F7899D41F3C3
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install yarn -y
```

Comment:
From what I know `Yarn` can be also installed per `NPM`.

```
npm i -g yarn
```

Should not need any extra sources for APT.